Explanation about running a rehearse in another cluster (#524)
openshift/ci-docs: content/en/docs/how-tos/contributing-openshift-release.md

target component repository after the changes would be merged. This provides pull request authors early feedback about
how config changes impact CI setup.

> #### Reheare a job on another cluster

Reviewer: ```suggestion
#### Rehearse a job on another cluster
```

> The [dynamic scheduler](https://docs.ci.openshift.org/docs/internals/dynamic-scheduling/) automatically defines the cluster on which the job will run on, which does not honor the `cluster` field. We can work around this by changing the `cluster` and the job `name` fields to manually create a cache miss in order for the `cluster` field to be honored.

Reviewer: ```suggestion
The [dynamic scheduler](https://docs.ci.openshift.org/docs/internals/dynamic-scheduling/) automatically defines the cluster on which the job will run, and does not always honor the `cluster` field. We can work around this by changing the `cluster` and the job `name` fields to manually create a cache miss in order for the `cluster` field to be honored.
```

> > _The `name` change is a temporary workaround only to run rehearsals and doesn't need to be merged after the tests._

Reviewer: ```suggestion
> _The `name` change is a temporary workaround only to run rehearsals and doesn't need to be merged after the rehearsals are completed._
```
